Quadriga/src/main/resources/locale/projectconstants.properties

```diff
@@ -6,4 +6,5 @@ dictionary_id.prefix=DICT
 concept_collection_id.prefix=CC
 workspace_id.prefix=WS
 network_id.prefix=NET
-notype_id.prefix=ID
\ No newline at end of file
+notype_id.prefix=ID
+projectblogentry_id.prefix=PROJBLOGENTRY

```

Comment:
I think I mentioned this a couple of times now. This should be only 3 or 4 characters long, e.g. BLGE.